src/attributes.js

```diff
@@ -29,7 +29,7 @@ var applyAttr = function(el, name, value) {
   if (value == null) {
     el.removeAttribute(name);
   } else {
-    el.setAttribute(name, value);
+    el.setAttribute(name, /**@type {boolean|number|string}*/(value));
```

Comment:
Would it be better to enumerate in the `@param`? I'm not at all a fan of the inline type comments.